beetsplug/lastgenre/__init__.py

```diff
@@ -224,12 +225,17 @@ def _resolve_genres(self, tags):
 
         # c14n only adds allowed genres but we may have had forbidden genres in
         # the original tags list
-        tags = [x.title() for x in tags if self._is_allowed(x)]
+        tags = [self._format_tag(x) for x in tags if self._is_allowed(x)]
 
         return self.config['separator'].as_str().join(
             tags[:self.config['count'].get(int)]
         )
 
+    def _format_tag(self, tag):
+        if self.config["title_case"].get():
```

Comment:
This is equivalent (using automatic bool coercion):
```suggestion
        if self.config["title_case"]:
```

Reply:
👌